### ISSUE TITLE
Docblock indentation 

### ISSUE THREAD

**Issue by tomasnorre** [rectorphp/rector#2976]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.7.2 |
| Installed as   | composer dependency                  |

After updating to rector v.0.7.2, I get "funny" indentations suggestions for some, not all docblocks.

```
     /*****************************
-     *
-     * General Helper Functions
-     *
-     *****************************/
-
-    /**
-     * This method is used to add a message to the internal queue
-     *
-     * @param  string $message the message itself
-     * @param  int $severity message level (-1 = success (default), 0 = info, 1 = notice, 2 = warning, 3 = error)
-     */
-    protected function addMessage(string $message, int $severity = FlashMessage::OK): void
+				 *
+				 * General Helper Functions
+				 *
+				 *****************************/
+				/**
+				 * This method is used to add a message to the internal queue
+				 *
+				 * @param   string $message the message itself
+				 * @param   int $severity message level (-1 = success (default), 0 = info, 1 = notice, 2 = warning, 3 = error)
+				 */
+				protected function addMessage(string $message, int $severity = FlashMessage::OK): void
     {
````

I haven't found a pattern in the behaviour yet, but perhaps you can point me into the correct direction. 

I'm honestly not sure if it's a problem with rector, or a misconfiguration on my site. 

I only run the dead-code set. 

## Expected Behaviour

I would expect the docblock to be "left" align as usual.

**Comment by TomasVotruba** [comment]
Try adding smalles demo possible in: https://getrector.org/demo

**Comment by tomasnorre** [comment]
At demo site, it's working like expected. 
https://getrector.org/demo/66cfe4f1-617b-4a1f-b180-a1b3b8996ad9#result

**Comment by tomasnorre** [comment]
You can see it here as well in the Travis CI, there is a little more "context"
https://travis-ci.org/AOEpeople/crawler/jobs/656789015

**Comment by tomasnorre** [comment]
I've done a little testing to see if I could locate the problem.

Regardless of which of the following sets I use it gets indented
```
dead-code
php70
php71
```

So looks like it hasn't something to do with a specific set.

**Comment by tomasnorre** [comment]
I've found the problem, but not the solution.

For some reasons the indentation is after the `vendor/bin/rector process --config rector-ci.yaml` is done with tabs instead of spaces.

The number of indentations are still correct but the character isn't.

**Comment by tomasnorre** [comment]
I don't know yet if can be related to following commit https://github.com/rectorphp/rector/commit/04c62b0c11a4f6c537d24a4cb8dfbe8ff919fc9d

But it as least has something to do with indentations and tabs too.

**Comment by TomasVotruba** [comment]
Failing demo is needed, it might need to modify the input file

**Comment by tomasnorre** [comment]
This could also have relevance
https://github.com/rectorphp/rector/commit/375eb2f57c61079665c4a842917294a50d19b7a4

**Comment by tomasnorre** [comment]
I'll see if I can make a failing demo.

**Comment by tomasnorre** [comment]
I have an example code now.
https://getrector.org/demo/eea02846-e17f-4e44-a6eb-7f9596a19134#result

After the 
```
final class DemoFile
{

```

I added a tab manually.. And that triggered the PHPDoc to indent with tabs instead of spaces, but only for the second PHPDoc Block.

**Comment by tomasnorre** [comment]
If I clean my file for all tab characters regardless where they are positioned. It works.

**Comment by alexanderschnitzler** [comment]
Hey guys, this looks like a duplicate of #2944 which has been solved a couple of days ago with https://github.com/rectorphp/rector/pull/2947.

**Comment by TomasVotruba** [comment]
@alexanderschnitzler Just released v0.7.3 with that fix

**Comment by tomasnorre** [comment]
I cannot test, as I fixed the issue in my files, but I'll close the the issue.
Thanks for taking care.
